projects/tegisto/index.js

```diff
@@ -0,0 +1,35 @@
+const { ethers } = require("ethers");
+const sdk = require("@defillama/sdk");
+
+// Tegisto stakes
+const stakeContractAddresses = {
+  kava: '0x744Dd9f79b80437a9e5eb0292128045F51C48b6d',
```

Comment:
I am confused as to what this contract is, it is not verified  nor does it hold any tegisto tokens, and even if it did, it does not count towards tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
Staking program is new right now. The reward and the first stake added right now.